test/integration/master_routes_test.go

```diff
@@ -255,15 +260,17 @@ func TestWellKnownOAuthOff(t *testing.T) {
 }
 
 var preferredVersions = map[string]string{
-	"":                          "v1",
-	"apps":                      "v1beta1",
+	"": "v1",
+	"admissionregistration.k8s.io": "v1beta1",
```

Comment:
gofmt is happy?

Reply:
We don't run gofmt on none of our tests, we should probably do. I'll open a PR fixing it.

Reply:
> gofmt is happy?

apparently. that may have been autofixed later.  I had some IDE troubles on the way through this.